thavalon-server/src/game/mod.rs

```diff
@@ -238,3 +239,25 @@ static FIVE_PLAYER: GameSpec = GameSpec {
     max_proposals: 5,
     double_fail_mission_four: false,
 };
+
+/// Two-player games, for testing
+static TWO_PLAYER: GameSpec = GameSpec {
+    players: 2,
+    mission_sizes: [2, 2, 2, 2, 2],
```

Comment:
Minor comment, but could we make it something like 1, 1, 2, 2, 2, so we could test having one player not on the mission?